frontend/www/js/omegaup/components/problem/List.vue

```diff
@@ -39,7 +39,7 @@
               <div class="tag-list"
                    v-bind:title="`${problem.tags.map(tag =&gt; tag.name).join(' ')}`"
                    v-if="problem.tags.length">
-                <a v-bind:class="tag.autogenerated === '1' ? 'tag-autogenerated':'tag'"
+                <a v-bind:class="!tag.autogenerated ? 'tag':'tag-autogenerated'"
```

Comment:
nit: intenta usar lógica positiva siempre que puedas

```suggestion
                <a v-bind:class="tag.autogenerated ? 'tag-autogenerated' : 'tag'"
```